### PR TITLE
Events: New ReadMe file

### PR DESCRIPTION
**Pull request by Postroutine**
Rewrite the README file and change to markdown

### CODE REVIEW

File: events/README.md [hunk 1]
```diff
@@ -1,53 +1,82 @@
-events ----
+Events plugin
+=============
+
+This plugin let you declare an event in your blog posts by using their
+metadatas. An Icalendar file is generated with all events.
```
Comment (justinmayer) [inline events/README.md]: > This plugin allows you to put events in your content via metadata. An iCal file is generated containing all events.

File: events/README.md [hunk 2]
```diff
@@ -1,53 +1,82 @@
...
+}
+```
 
-Create articles and usual and add the "event-start" metadata to turn
-them into events. The event start is independent of the article "date".
-"event-start" is in "YYYY-MM-DD hh:mm" format. Also add "event-end", in
-the same format, or "event-duration" as a number followed by a
-dimension:
+Settings:
+- `ics_fname`: Where the Icalendar file is written
```
Comment (justinmayer) [inline events/README.md]: Icalendar → iCal

File: events/README.md [hunk 3]
```diff
@@ -1,53 +1,82 @@
...
 
-w: weeks d: days h: hours m: minutes s: seconds
 
-You can also specify an optional "location"
+Usage
+-----
 
-Example in ReST format:
+When you write an article, you can use theses metadatas
```
Comment (justinmayer) [inline events/README.md]: > You can use the following metadata in your content:

File: events/README.md [hunk 4]
```diff
@@ -1,53 +1,82 @@
...
+- `event-start`: When the event will start in "YYYY-MM-DD hh:mm"
+- `event-end`: When the event will stop in "YYYY-MM-DD hh:mm"
+- `event-duration`: How many times the event will continue [1]
```
Comment (justinmayer) [inline events/README.md]: > - `event-duration`: The duration of the event [1]

File: events/README.md [hunk 5]
```diff
@@ -1,53 +1,82 @@
...
+- `event-location`: Where the event take place
```
Comment (justinmayer) [inline events/README.md]: - `event-location`: Where the event takes place

File: events/README.md [hunk 6]
```diff
@@ -1,53 +1,82 @@
...
+- `h`: hours
+- `m`: minutes
+- `s`: seconds
 
-To generate an sorted event list in a dedicated page copy the
-events\_list.html template under the templates directory in your theme,
-then create a page:
 
-content/pages/events\_list.rst:
+Exemples
```
Comment (justinmayer) [inline events/README.md]: Exemples → Examples

File: events/README.md [hunk 7]
```diff
@@ -1,53 +1,82 @@
...
+--------
 
-    Events list
-    ###########
-    :slug: events-list
-    :summary:
-    :template: events_list
+Example in ReST format:
```
Comment (justinmayer) [inline events/README.md]: ReST → reST

File: events/README.md [hunk 8]
```diff
@@ -1,53 +1,82 @@
...
+```ReST
```
Comment (justinmayer) [inline events/README.md]: ReST → reST

File: events/README.md [hunk 9]
```diff
@@ -1,53 +1,82 @@
...
+Event-start: 2015-01-21 10:30
+Event-duration: 2h
+Event-location: somewhere
+```
+
+
+Dedicated page
+--------------
+
+To generate an sorted event list in a dedicated page:
```
Comment (justinmayer) [inline events/README.md]: > To generate a sorted event list in its own dedicated page:

File: events/README.md [hunk 10]
```diff
@@ -1,53 +1,82 @@
...
+- Copy the `events_list.html` template under the templates directory of your theme
+- Create a page for this list, for exemple in `content/pages/events_list.rst`
```
Comment (justinmayer) [inline events/README.md]: exemple → example

File: events/README.md [hunk 11]
```diff
@@ -1,53 +1,82 @@
...
+- Write these metadatas in your page:
```
Comment (justinmayer) [inline events/README.md]: Include the following metadata in your content:

File: events/README.md [hunk 12]
```diff
@@ -1,53 +1,82 @@
...
+```ReST
```
Comment (justinmayer) [inline events/README.md]: ReST → reST

**Comment by Postroutine** [comment]
Modifications done. 
I hope I don't forget anything.

**Comment by justinmayer** [comment]
Many thanks for your contribution, @Postroutine.  (^_^)

**Pull request merged**